### PR TITLE
kitchen counter Interaction

### PR DESCRIPTION
**Pull request by Tanisha800**
if you like the interaction please add it to hacktoberfest2024

### CODE REVIEW

**Review by r4pt0s — CHANGES_REQUESTED** [review]
Hi @Tanisha800,
Your PR looks like a duplicate of #186
Even the comment in the pull request.

Can you change the behavior of your implementation here a bit?

File: src/interactions/map_campus_house_1/kitchen_counter.interaction.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import { time } from '../../kplayCtx';
 import { displayDialogue, displayPermissionBox } from '../../utils';
 import { updateEnergyState } from '../../utils/energyUpdate';
 
-export const kitchencounterInteractions = (player, k, map) => {
-    player.onCollide('kitchen_slab', async () => {
```
Comment (Tanisha800) [inline src/interactions/map_campus_house_1/kitchen_counter.interaction.js]: kitchen_slab must be change to kitchen_counter

**Review by Tanisha800 — COMMENTED** [review]
commited the changes

**Comment by r4pt0s** [comment]
@Tanisha800 there is still a merge conflict. Can you please fix it 🙏🏻?

**Comment by r4pt0s** [comment]
Closed due to inactivity

**Pull request closed without merging**